### DISCUSSION TITLE
Select options not visible in modal (v6)

### DISCUSSION THREAD

**Discussion by Rigel772** [mantinedev/mantine#3682] [Feedback]
Please take a look here:
https://codesandbox.io/s/mantine-modal-select-kw2f3s?file=/src/App.tsx

**Comment by rtivital** [comment]
They are visible in your sandbox

**Reply by Rigel772** [reply]
True, I have found out I need to add "withinPortal" for select to work, it is not working by default, check the sandbox now.
If it is expected behavior I think should be documented... or withinPortal should be true by default.
